flink-table/flink-table-common/src/main/java/org/apache/flink/table/api/UnresolvedException.java

```diff
@@ -18,7 +18,10 @@
 
 package org.apache.flink.table.api;
 
+import org.apache.flink.annotation.PublicEvolving;
+
 /** Exception for unwanted method calling on unresolved expression. */
+@PublicEvolving
 public class UnresolvedException extends RuntimeException {
```

Comment:
This exception is referenced only in one place in the planner, so perhaps move it to the planner without marking it public evolving?